### PR TITLE
Is perms and more

### PR DESCRIPTION
**Pull request by Antony1060**
Changes:
- /is perms
- /is paypal
- /is setpaypal
- /is who/info
- /sb checkpaypal
- SavageFTOP support
- island_name PAPI placeholder
- IslandChatEvent
- IslandChestMoveEvent

### CODE REVIEW

**Comment by Antony1060** [comment]
Also fixed events

File: src/main/kotlin/net/savagelabs/skyblockx/core/Island.kt [hunk 1]
```diff
@@ -606,6 +635,16 @@ fun getIslandFromLocation(location: Location): Island? {
     return null
 }
 
+// This is the same as getIslandFromLocation but it returns an island if the "location" is "amount" of blocks outside of said island
+fun getIslandFromLocationWide(location: Location, amount: Int): Island? {
```
Comment (lindrax) [inline src/main/kotlin/net/savagelabs/skyblockx/core/Island.kt:639]: Could be narrowed down to a one-liner with precisely the same speed.

File: src/main/kotlin/net/savagelabs/skyblockx/event/IslandChatEvent.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.bukkit.event.Cancellable
+import org.bukkit.event.HandlerList
+import org.bukkit.event.player.PlayerEvent
+
+class IslandChatEvent(player: Player, val island: Island, val message: String) : PlayerEvent(player), Cancellable {
+
+    private var cancelled = false
+
+    companion object {
+        private val HANDLERS = HandlerList()
```
Comment (lindrax) [inline src/main/kotlin/net/savagelabs/skyblockx/event/IslandChatEvent.kt:14]: Can be narrowed down to:
```
@JvmStatic
val handlerList = HandlerList()
```

File: src/main/kotlin/net/savagelabs/skyblockx/core/Island.kt (hunk 1 above)
Comment (Antony1060) [reply src/main/kotlin/net/savagelabs/skyblockx/core/Island.kt:639]: I know now, just, when I was making the method, I didn't. Thank you though.

File: src/main/kotlin/net/savagelabs/skyblockx/event/IslandChatEvent.kt (hunk 2 above)
Comment (Antony1060) [reply src/main/kotlin/net/savagelabs/skyblockx/event/IslandChatEvent.kt:14]: Thank you <3

**Pull request closed without merging**